README.md

```diff
@@ -127,10 +127,23 @@ pyenv/bin/py.test -vv --tb=short tests
 
 ### Development
 
-Provided you installed everything with `tox`, you should be able to run a development server with
+Provided you installed everything with `tox`, you should be able to run a development server as follows:
+
+First set up the environment variables that do not default to usable values. 
+Obtain the user ID's and passwords necessary for the two databases from PCIC IT.
+We typically use port 8000 but any port will do.
+
+```bash
+export DSN=postgresql://<USER>:<PASSWORD>@db3.pcic.uvic.ca/pcic_meta
+export DATA_ROOT=http://127.0.0.1:<PORT>/data
+export PCDS_DSN=postgresql://<USER>:<PASSWORD>@db3.pcic.uvic.ca/crmp
+export APP_ROOT=http://127.0.0.1:<PORT>
+```
```

Comment:
This is probably outside the scope of this PR, but I feel there should be a better way to organize the README. There's a lot of information in it, and it's not intuitively organized. There's a section on environmental variables that describes them in general terms, and then down here there's specific options for them. 

I remember starting with this project and being completely unsure which of the many instructions in various sections applied to what I was trying to do.

Reply:
https://github.com/pacificclimate/pdp/issues/118